doc/host-setup.md

```diff
@@ -146,3 +146,39 @@ Checking the current value again will reveal that the helper is now disabled
 # cat /sys/module/drbd/parameters/usermode_helper
 disabled
 ```
+
+## Disable Lvmetad in CentOS 7 and Ubuntu 18.04
+
+Reference: https://access.redhat.com/solutions/2053483
+
+CentOS 7 and Ubuntu 18.04 by default use **lvmetad**,  a metadata caching daemon for LVM. The daemon receives notifications from udev rules. Through these notifications, lvmetad has an up-to-date and consistent image of the volume groups available in the system. lvmetad improves performance of LVM commands by saving them from scanning the disks in every execution. 
+
+`piraeus-ns-node` pod runs lvm commands inside a container without using any lvmetad or udev. And it does not communicate to lvmetad daemon in the operation system. Therefore, users must run `pvscan --cache` in the OS each time to view LVM changes after a Piraeus volume operation. 
+
+In CentOS 8 and in Ubuntu 20.04 (focal), lvmetad is deprecated because lvm2 has removed it since version 2.03. **Therefore, it is also recommended to disable lvmetad in CentOS 7 and Ubuntu 18.04 binoic**. 
+
+Follow below steps to disable lvmetad completely: 
+```bash
+$ systemctl stop lvm2-lvmetad.socket lvm2-lvmetad.service
+
+$ systemctl disable lvm2-lvmetad.service
+
+$ systemctl mask lvm2-lvmetad.socket
```

Comment:
Why to mask without disabling lvm2-lvmetad.socket?

Reply:
It is also recommend by RedHat's article "[https://access.redhat.com/solutions/2053483] https://access.redhat.com/solutions/2053483"